src/launchWizard/launchWizard.js

```diff
@@ -27,15 +27,88 @@ function getConfigIndex() {
   if (configSelectedValue === 'New Config') {
     document.getElementById('nameLabel').style =
       'margin-top: 10px; visibility: visible;'
+    document.getElementById('copyLaunchConfigButton').style =
+      'visibility: hidden;'
   } else {
     document.getElementById('nameLabel').style = 'visibility: hidden;'
+    document.getElementById('copyLaunchConfigButton').style =
+      'visibility: visible;'
   }
 
   return configSelectedValue === 'New Config'
     ? -1
     : configSelectionBox.selectedIndex
 }
 
+function getConfigValues() {
+  var configSelectionBox = document.getElementById('configSelected')
+  var configSelectedValue =
+    configSelectionBox.options[configSelectionBox.selectedIndex].value
+  const name =
+    configSelectedValue === 'New Config'
+      ? document.getElementById('name').value
+      : configSelectedValue
+  const data = document.getElementById('data').value
+  const debugServer = parseInt(document.getElementById('debugServer').value)
+  const infosetFormat = document.getElementById('infosetFormat').value
+  const infosetOutputFilePath = document.getElementById(
+    'infosetOutputFilePath'
+  ).value
+  const infosetOutputType = document.getElementById('infosetOutputType').value
+  const tdmlAction = document.getElementById('tdmlAction').value
+  const tdmlName = document.getElementById('tdmlName').value
+  const tdmlDescription = document.getElementById('tdmlDescription').value
+  const tdmlPath = document.getElementById('tdmlPath').value
+  const openHexView = document.getElementById('openHexView').checked
```

Comment:
Are hexviews still being used? I remember removing the `hexy` capability from the old data editor and don't recall seeing it being used anywhere else throughout the project. Granted I haven't been thoroughly searching.

Reply:
Yeah the hexview is still being used but believe omega-edit over takes it. I will make an issue to remove the hex view stuff as it will be replaced by omega-edit.